packages/specs/schema/test/integrations/generics.pageable.integration.spec.ts

```
@@ -0,0 +1,815 @@
+  Returns,
+  SpecTypes,
+  string
+} from "../../src/index.js";
+import {validateSpec} from "../helpers/validateSpec.js";
+
+export class Pageable {
+  @Integer()
+  @Min(0)
+  @Default(0)
+  @Description("Page number.")
+  page: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)